test/test_extensions.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class TestExtensions < Minitest::Test
+  def setup
+    @markdown = <<-MD
+One extension:
+
+| a   | b   |
+| --- | --- |
+| c   | d   |
+
+Another extension:
+
+~~hi~~
+    MD
+  end
+
+  def test_uses_specified_extensions
+    none_in_use = CommonMarker.render_html(@markdown, :default, %i[])
+    assert none_in_use.include?("| a")
+    assert none_in_use.include?("~~hi~~")
+
+    none_in_use = CommonMarker.render_html(@markdown, :default, %i[table])
+    refute none_in_use.include?("| a")
+    assert none_in_use.include?("~~hi~~")
+
+    none_in_use = CommonMarker.render_html(@markdown, :default, %i[strikethrough])
+    assert none_in_use.include?("| a")
+    refute none_in_use.include?("~~hi~~")
+
+    none_in_use = CommonMarker.render_html(@markdown, :default, %i[table strikethrough])
+    refute none_in_use.include?("| a")
+    refute none_in_use.include?("~~hi~~")
```

Comment:
Could we get some assertions on the expected HTML, for example, `<strike>hi</strike>` or `<td>a</td>`? (For these, and the tests above.)

Also, could you demonstrate what happens with something like, asterisks in a table cell? `github-markdown` does support this at the moment.